window.go

```diff
@@ -54,8 +56,10 @@ const (
 	EventNameWindowEventUnresponsive                  = "window.event.unresponsive"
 	EventNameWindowEventDidGetRedirectRequest         = "window.event.did.get.redirect.request"
 	EventNameWindowEventWebContentsExecutedJavaScript = "window.event.web.contents.executed.javascript"
+	EventNameWindowEventWebContentsSetProxy           = "window.event.web.contents.set.proxy"
 	EventNameWindowEventWillNavigate                  = "window.event.will.navigate"
 	EventNameWindowEventUpdatedCustomOptions          = "window.event.updated.custom.options"
+	EventNameWindowLoadUrl                            = "window.event.load.url"
```

Comment:
Value should be `window.event.load.url` and name should be `EventNameWindowLoadURL` (`url` is all capital letters)